### ISSUE TITLE
Bad request for Google Drive login

### ISSUE THREAD

**Issue by bit-man** [MasterSoft24/CloudCross#84]
Installed CloudCross and when accessing link for Google drive login returns bad request

```
$ ccross -a --provider google
"-------------------------------------"
"Please go to this URL and confirm application credentials"
"https://accounts.google.com/ServiceLogin?continue=https%3A%2F%2Faccounts.google.com%2Fsignin%2Foauth%2Flegacy%2Fconsent%3Fauthuser%3Dunknown%26part%3Dxxxxxxxxxxxxxxx&sacu=1&oauth=1&rip=1"

$ ccross --version
CloudCross  v 1 . 4 . 6  for Linux

$ uname -a
Linux mintux 4.19.0-9-amd64 #1 SMP Debian 4.19.118-2+deb10u1 (2020-06-07) x86_64 GNU/Linux
```

**Comment by cassiodoroVicinetti** [comment]
I can confirm this issue.

```
$ uname -a
Linux luca-debian 5.7.0-1-amd64 #1 SMP Debian 5.7.6-1 (2020-06-24) x86_64 GNU/Linux

$ ccross --version
CloudCross  v 1 . 4 . 6  for Linux
```

![Schermata 2020-07-22 alle 10 23 36-fullpage](https://user-images.githubusercontent.com/1621526/88153435-b3b35a80-cc05-11ea-8334-68160a33a7d3.png)

**Comment by MasterSoft24** [comment]
fixed by the last 1.4.7 release
